### DISCUSSION TITLE
【委托】救命！我被困在空洞里了！传奇绳匠请救救我！

### DISCUSSION THREAD

**Discussion by Lululu-123** [share121/inter-knot#1890] [其他]
![20240813-171903](https://github.com/user-attachments/assets/33818ce3-f9d6-4868-9b4c-50265b2c147b)

**Comment by 834398220** [comment]
这个委托我接了

**Comment by Lululu-123** [comment]
既然是绳匠出马，那我就放心了

**Comment by Lululu-123** [comment]
![Uploading 356924823-f65c7a73-8993-414c-8f0e-e315e26efd4e.gif…]()

**Comment by CloudSwordSage** [comment]
嘶，这个委托不好打啊，得**加钱**
---
![image](https://github.com/user-attachments/assets/9b2f10d8-2afa-45e2-80cb-972baff9367d)

**Comment by SednaMi** [comment]
巴莱大厦里面？最近不是空洞活跃下降了？助lz好运🤞

**Comment by duomachan** [comment]
得加钱

**Comment by Lululu-123** [comment]
没钱！

**Comment by cleo66** [comment]
![640 (4)](https://github.com/user-attachments/assets/6182405d-fa0d-4208-a7c8-c07c8d054564)

**Comment by koutianhhh** [comment]
有绳匠在我就放心了

**Comment by Momo520v** [comment]
这个委托我接了

**Reply by fengkong-cjb** [reply]
这个委托我接了

**Comment by 129776** [comment]
这个委托他们接了
![Uploading Screenshot_20241226_220412.jpg…]()

**Comment by Kiana175** [comment]
这个委托本小姐接定了！

**Comment by zhongliogyour** [comment]
委托交给我，我可以最快的速度帮你完成，因为我是维多利亚家政！

**Comment by tanchideshaohuaya** [comment]
这个委托我接了

**Comment by bxzwq** [comment]
这个委托我接了

**Comment by 13586-xgp** [comment]
这个委托我接了

**Comment by Alice-warriors** [comment]
不兑吧 你这是切进后室的level1的子层级红室了吧 没救了 下一个

**Comment by Kazuto-Kiritani** [comment]
一看就是巴莱大厦，这个委托我接了

**Comment by jiuqi001** [comment]
接了 委托费结一下

**Comment by xudaz** [comment]
这
